apps/reference/docs/guides/auth/auth-ui/auth-ui-customization.mdx

```diff
@@ -0,0 +1,254 @@
+---
+id: auth-ui-customization
+title: Customization
+description: Securing your user data with Row Level Security.
+---
+
+# Customizing Auth UI
+
+Auth UI is a React component that provides a UI that 'works out of the box'.
+
+There are several ways to customize Auth UI:
+
+- Using one of the [predefined Themes](#themes) that comes with Auth UI.
+- Extending a theme by [overriding the variable tokens](#overwriting-themes) in a theme.
+- [Creating your own Theme](#creating-your-own-theme)
+- [Using your own CSS classes](#using-your-own-css-classes)
+- [Using inline styles](#using-your-own-inline-styles)
+- [Using your own labels](#using-your-own-custom-labels)
+
+Before proceeding, you may want to already be familiar with the [installation of Auth UI](/docs/guides/auth/auth-ui/auth-ui-install).
```

Comment:
Can we roll both of these pages into a single guide? 

`/docs/guides/auth/auth-ui`

Better than sending them back to another page

Reply:
ha, i was actually going to be adding another page for upgrade guide 🙈